cli/src/test/resources/ide-projects/settings/project/readme

```diff
@@ -0,0 +1 @@
+this is the IDE_HOME directory
```

Comment:
This file can be removed. Only required if we want to read from it in a test. Please use `.gitkeep` instead if a folder is needed.